From fc1ac8bef8dcc3194857ecd32519aca4867b4fa1 Mon Sep 17 00:00:00 2001
From: Bas Couwenberg <sebastic@xs4all.nl>
Date: Tue, 23 Feb 2021 18:40:14 +0100
Subject: [PATCH] Add support for PROJ 8.0.0 in FindProj.cmake.

proj_api.h is no longer available in PROJ 8.0.0, use proj.h instead.

Fixes: #41735
---
 cmake/FindProj.cmake | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cmake/FindProj.cmake b/cmake/FindProj.cmake
index 0afee15fd14c..ccfaf13d9edf 100644
--- a/cmake/FindProj.cmake
+++ b/cmake/FindProj.cmake
@@ -40,6 +40,12 @@ FIND_PATH(PROJ_INCLUDE_DIR proj_api.h
   "$ENV{INCLUDE}"
   "$ENV{LIB_DIR}/include"
   )
+IF (NOT PROJ_INCLUDE_DIR)
+  FIND_PATH(PROJ_INCLUDE_DIR proj.h
+    "$ENV{INCLUDE}"
+    "$ENV{LIB_DIR}/include"
+    )
+ENDIF (NOT PROJ_INCLUDE_DIR)
 
 FIND_LIBRARY(PROJ_LIBRARY NAMES proj_i proj PATHS
   "$ENV{LIB}"
